### PR TITLE
SWC-6041: use the same view for the challenge participants widget as …

### PR DESCRIPTION
**Pull request by jay-hodgson**
…the team members widget

### CODE REVIEW

File: src/main/java/org/sagebionetworks/web/client/PortalGinInjector.java [hunk 1]
```diff
@@ -175,7 +175,7 @@
 import org.sagebionetworks.web.client.widget.entity.renderer.TIFFPreviewWidget;
 import org.sagebionetworks.web.client.widget.entity.renderer.TableOfContentsWidget;
 import org.sagebionetworks.web.client.widget.entity.renderer.TeamMemberCountWidget;
-import org.sagebionetworks.web.client.widget.entity.renderer.TeamMemberRowWidget;
+import org.sagebionetworks.web.client.widget.entity.renderer.UserListRowWidget;
```
Comment (jay-hodgson) [inline src/main/java/org/sagebionetworks/web/client/PortalGinInjector.java:178]: renamed TeamMemberRowWidget to UserListRowWidget

File: src/main/java/org/sagebionetworks/web/client/PortalGinModule.java [hunk 2]
```diff
@@ -1243,8 +1241,7 @@ protected void configure() {
 		bind(TableQueryResultWikiWidgetView.class).to(TableQueryResultWikiWidgetViewImpl.class);
 
 		bind(SingleButtonView.class).to(SingleButtonViewImpl.class);
-		bind(UserListView.class).to(UserListViewImpl.class);
```
Comment (jay-hodgson) [inline src/main/java/org/sagebionetworks/web/client/PortalGinModule.java:1246]: Deleted old UserListView (was only used by the challenge participant list widget)

File: src/main/java/org/sagebionetworks/web/client/PortalGinModule.java [hunk 3]
```diff
@@ -1405,8 +1402,8 @@ protected void configure() {
 		bind(SelfSignAccessRequirementWidgetView.class).to(SelfSignAccessRequirementWidgetViewImpl.class);
 		bind(SubjectWidgetView.class).to(SubjectWidgetViewImpl.class);
 		bind(AwsLoginView.class).to(AwsLoginViewImpl.class);
-		bind(TeamMemberRowWidgetView.class).to(TeamMemberRowWidgetViewImpl.class);
-		bind(TeamMembersWidgetView.class).to(TeamMembersWidgetViewImpl.class);
+		bind(UserListRowWidgetView.class).to(UserListRowWidgetViewImpl.class);
+		bind(UserListView.class).to(UserListViewImpl.class);
```
Comment (jay-hodgson) [inline src/main/java/org/sagebionetworks/web/client/PortalGinModule.java:1406]: Renamed TeamMembersWidgetView to UserListView

File: src/main/java/org/sagebionetworks/web/client/widget/entity/renderer/ChallengeParticipantsWidget.java [hunk 4]
```diff
@@ -1,46 +1,55 @@
...
-	private Callback widgetRefreshRequired;
 	private BasicPaginationWidget paginationWidget;
+	private PortalGinInjector ginInjector;
+	private SynapseAlert synAlert;
 	public static final Long DEFAULT_PARTICIPANT_LIMIT = 50L;
 	public static final Long DEFAULT_OFFSET = 0L;
 
 	@Inject
-	public ChallengeParticipantsWidget(UserListView view, BasicPaginationWidget paginationWidget, ChallengeClientAsync synapseClient) {
+	public ChallengeParticipantsWidget(UserListView view, BasicPaginationWidget paginationWidget, ChallengeClientAsync synapseClient, SynapseAlert synAlert, PortalGinInjector ginInjector) {
```
Comment (jay-hodgson) [inline src/main/java/org/sagebionetworks/web/client/widget/entity/renderer/ChallengeParticipantsWidget.java:38]: Both the TeamMembersWidget and this ChallengeParticipantsWidget now use the same view, UserListView

**Review by nickgros — APPROVED** [review]

**Pull request merged**